services/app-graphql/src/queries/indexHealthPlanPackages.graphql

```diff
@@ -22,9 +22,13 @@ query indexHealthPlanPackages {
                         createdAt
                         unlockInfo {
                             updatedAt
+                            updatedBy
```

Comment:
I made changes here to  be able to use the shared frontend utility `getCurrentRevisionFromHealthPlanPackage`. Want to avoid domain to base 64  calls in React code, we should be trying to isolate that error handling to one place.